iconservice/iconscore/icon_system_score_base.py

```diff
@@ -110,3 +119,14 @@ def disqualify_prep(self, address: 'Address') -> Tuple[bool, str]:
             reason = str(e)
         finally:
             return success, reason
+
+    def apply_revision_change(self, revision):
```

Comment:
Is this method public?
Is it possible to call this explicitly in Governance score?

Reply:
Yes, called in governance SCORE when revision network proposal approved